lib/siwapp_web/live/invoices_live/index.ex

```diff
@@ -143,6 +149,47 @@ defmodule SiwappWeb.InvoicesLive.Index do
     {:noreply, socket}
   end
 
+  def handle_event("reorder", %{"field" => field, "order" => order}, socket) do
+    order_by =
+      order
+      |> String.to_atom()
+      |> Searches.prepare_order_by(String.to_atom(field))
+
+    socket =
+      socket
+      |> assign(:order_by, order_by)
+      |> assign(:sort_map, %Sortener{order: order, field: field})
+      |> assign(
+        :invoices,
+        Searches.filters(socket.assigns.query,
+          limit: @invoices_limit,
+          preload: [:series],
+          deleted_at_query: true,
+          order_by: order_by
+        )
+      )
+
+    {:noreply, socket}
+  end
+
+  def handle_event("reorder", _params, socket) do
```

Comment:
To avoid repeating code here yo can do:
```
  def handle_event("reorder", _params, socket) do
    handle_event("reorder", %{"field" => "id", "order" => "desc"}, socket)
  end
```

Reply:
Yep, very well seen :+1: